app/src/main/java/com/example/partyPlayer/TrackService.kt

```diff
@@ -0,0 +1,6 @@
+package com.example.partyPlayer
+
+class TrackService {
+
+    // fun fetchTracks(artistName)
```

Comment:
added space after the comment // in accordance with horizontal spacing conventions: 
https://kotlinlang.org/docs/reference/coding-conventions.html#formatting